google/datalab/bigquery/commands/_bigquery.py

```diff
@@ -719,12 +719,14 @@ def _table_cell(args, cell_body):
       if args['project'] is None:
         datasets = [google.datalab.bigquery.Dataset(args['dataset'])]
       else:
-        datasets = [google.datalab.bigquery.Dataset((args['project'], args['dataset']))]
+        context = google.datalab.Context(args['project'],
```

Comment:
I don't fully understand this code, but this code branch is only executed when the user has specified both project and dataset, yet it looks like in the updated code you no longer use the dataset arg. If this is correct, it probably warrants a comment. Or maybe even throw an exception if the user specifies both and we are going to ignore the dataset arg.